File: day36.cpp


// C++ Code Example
#include <iostream>
using namespace std;

int dailycoding (int x)
{
  int output;
  output = x * 2;
  return output;
}

int main ()
{
  cout << dailycoding(3);
}

// Golang Example
package main

import "fmt"

func dailycoding(x int) int {
	output := x * 2
	return output
}

func main() {
	fmt.Println(dailycoding(3))
}


// JavaScript Example
function dailycoding(x) {
	output = x * 2;
	return output;
}
dailycoding(3);
// Output 6


// PHP Example
function dailycoding($input) {
	$output = $input * 2; 
	return $output; 
}
echo dailycoding(3);
// Output 6
	

File: day36.go
package main

import (
	"fmt"
)

func dailycoding(num ...int) {
	num[0] = 1
}
func main() {
	i := []int{2, 3, 6}
	dailycoding(i...)
	fmt.Println(i[1] + i[0])
}

// 4 (3+1)


function number_double($input) {
	$output = $input * 2; //double the input value
	echo $output; 
	//print display $output result
}
echo number_double(2);
	